firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/FirebaseInAppMessaging.java

```diff
@@ -336,6 +336,16 @@ public void removeDisplayErrorListener(
     developerListenerManager.removeDisplayErrorListener(displayErrorListener);
   }
 
+  /**
+   * Unregisters a dismiss listener.
+   *
+   * @param dismissListener
```

Comment:
Would more info about the param be helpful?

Reply:
Done